locales/es-419.json

```diff
@@ -1 +1,195 @@
+    },
+    "delete_posts": {
+      "title": "¿De verdad quieres eliminar esta publicación?"
+    },
+    "mute_account": {
+      "title": "¿De verdad quieres silenciar a {0}?"
+    },
+    "show_reblogs": {
+      "cancel": "No",
+      "confirm": "Sí, ver",
+      "title": "¿De verdad quieres ver los retoots de {0}"
+    },
+    "unfollow": {
+      "title": "¿De verdad quieres dejar de seguir?"
+    }
+  },
+  "error": {
+    "file_size_cannot_exceed_n_mb": "El tamaño del archivo no puede ser de más de {0}MB",
+    "unsupported_file_format": "Formato de archivo no soportado"
+  },
+  "help": {
+    "desc_highlight": "Es normal que aparezcan algunos errores y funcionalidades que aún estén en desarrollo.",
+    "desc_para1": "¡Gracias por tu interés en probar Elk, nuestro cliente genérico en desarrollo para Mastodon!",
+    "desc_para2": "Estamos haciendo lo posible para ir mejorando constantemente.",
+    "desc_para4": "Elk es de código abierto. Si quieres probar para ayudar, opinar o contribuir,",
+    "desc_para5": "contáctanos a través de GitHub"
+  },
+  "list": {
+    "add_account": "Añadir cuenta a la lista",
+    "remove_account": "Quitar cuenta de la lista"
+  },
+  "menu": {
+    "block_domain": "Ocultar dominio {0}",
+    "delete_and_redraft": "Eliminar y volver a borrador",
+    "edit": "Actualizar",
+    "pin_on_profile": "Anclar en tu perfil",
+    "show_favourited_and_boosted_by": "Ver quien marcó como favorita y quien retooteó",
+    "show_reblogs": "Ver retoots de {0}",
+    "unblock_domain": "Ver dominio {0}",
+    "unmute_account": "Quitar silencio a {0}",
+    "unmute_conversation": "Quitar silencio de la publicación",
+    "unpin_on_profile": "Desanclar del perfil"
+  },
+  "nav": {
+    "back": "Atrás",
+    "blocked_domains": "Dominios ocultos",
+    "built_at": "Generado {0}",
+    "conversations": "Mensajes directos",
+    "favourites": "Favoritas",
+    "federated": "Historia federada",
+    "local": "Historia local",
+    "settings": "Preferencias",
+    "toggle_theme": "Cambiar tema de color",
+    "zen_mode": "Modo sin distracciones"
+  },
+  "notification": {
+    "followed_you": "te siguió",
+    "update_status": "actualizó su publicación"
+  },
+  "placeholder": {
+    "default_1": "¿En qué piensas?"
+  },
+  "search": {
+    "search_empty": "No se encontraron resultados para la búsqueda"
+  },
+  "settings": {
+    "about": {
+      "built_at": "Compilado el",
+      "sponsor_action": "Patrocina"
+    },
+    "account_settings": {
+      "description": "Actualiza los ajustes de tu cuenta en la interfaz de Mastodon.",
+      "label": "Configuración de cuenta"
+    },
+    "interface": {
+      "color_mode": "Temas de color",
+      "dark_mode": "Tema oscuro",
+      "default": " (predeterminado)",
+      "font_size": "Tamaño de fuente",
+      "light_mode": "Tema claro",
+      "system_mode": "Color del sistema"
+    },
+    "language": {
+      "display_language": "Idioma en pantalla",
+      "translations": {
```

Comment:
`translations` entry should be included in base `es` file

Reply:
It is included, actually

Reply:
on main, it is missing from `es` file

Reply:
![imagen](https://user-images.githubusercontent.com/6311119/216093545-0a204963-907d-4335-b049-8dab320bcba3.png)

Reply:
included here?